src/components/Header.jsx

```diff
@@ -0,0 +1,75 @@
+import "./Header.css";
+import { NavLink } from "react-router-dom";
+import { BiSolidCameraMovie } from "react-icons/bi";
+import { useState } from "react";
+
+export const Header = () => { 
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  // Navigation links data
+  const navLinks = [
+    { path: "/", label: "Now Playing" },
+    { path: "/popular", label: "Popular" },
+    { path: "/top-rated", label: "Top Rated" },
+    { path: "/upcoming", label: "Upcoming" }
+  ];
+  
```

Comment:
Love what you´ve done with the navbar and toggle dropdown on mobile. Looks very good!